packages/backend-common/src/scm/git.ts

```diff
@@ -76,12 +76,15 @@ export class Git {
     return git.commit({ fs, dir, message, author, committer });
   }
 
+  // https://isomorphic-git.org/docs/en/clone
```

Comment:
Replace `//` comments with a docstrings `/**`, that will also remove the undocumented warning in api-report :)

Reply:
👍 I've switched these all over